### PR TITLE
Rename `.vcApiTestImplementationsConfig.cjs` to `.localImplementationsConfig.cjs`.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Review by BigBlueHat — APPROVED** [review]
Naming is hard, but this one is definitely better!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # vc-test-suite-implementations Changelog
 
+## 3.0.0 - 2023-12-27
+
+### Changed
+- **BREAKING**: Rename `vcApiTestImplementationsConfig.cjs` config for testing
+  localhost implementations to `localImplementationsConfig.cjs`.
```
Comment (BigBlueHat) [inline CHANGELOG.md:7]: These both need `.` prefixes.

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
One small change, actually.

File: CHANGELOG.md (hunk 1 above)
Comment (JSAssassin) [reply CHANGELOG.md]: @BigBlueHat Good catch, thanks! Fixed here [6d1fae3](https://github.com/w3c-ccg/vc-test-suite-implementations/pull/98/commits/6d1fae36b9c6a11a2aef07ec40dcd7d2d685ce63)

**Review by BigBlueHat — APPROVED** [review]

**Pull request merged**